specification/cognitiveservices/data-plane/Language/preview/2021-11-01-preview/analyzeconversations.json

```diff
@@ -100,10 +198,6 @@
           "type": "string",
           "description": "The language to use in this request. This will be the language setting when communicating with all other target projects."
         },
-        "verbose": {
-          "type": "boolean",
-          "description": "If true, the service will return more detailed information in the response."
-        },
         "isLoggingEnabled": {
```

Comment:
This wasn't added in this PR, but just wanted to also mention that we have a similar parameter in TA called `loggingOptOut` which we actually pass in the task parameters object.  Based on the description here it looks like maybe it means the same thing but I could be wrong.  

Not super high priority, but maybe we can discuss unification of this at some point.

Reply:
That's a good idea, but for this release, I stick to isLoggingEnabled because it has the opposite semantic of LoggingOptOut.